chatbot-ui/components/ErrorSummary.tsx

```diff
@@ -0,0 +1,24 @@
+// TODO: Improve this component (I have only naively copied across what was already in app)
+// I don't think this needs to be a list, nor an anchor tag. Or alternatively we take an actual list of errors in the props.
```

Comment:
This code came from [Error summary](https://design-system.service.gov.uk/components/error-summary/) so I think it is correct, although clicking the link "Enter a question" in the error summary doesn't leave focus in the field. Maybe it should?

Reply:
Yes, the code is correct. When we do the full version it should behave as it does on EES but for the prototype I think it's fine to leave it hard coded to only accept one error and to not have the correct focus behaviour. I've removed the comment.